plugins/chat/assets/javascripts/discourse/components/chat-composer.js

```diff
@@ -304,6 +321,44 @@ export default Component.extend(TextareaTextManipulation, {
     this.onValueChange?.(this.value, this._uploads, this.replyToMsg);
   },
 
+  @bind
+  _debouncedCaptureMentions() {
+    this.mentionsTimer = debounce(
+      this,
+      this._captureMentions,
+      MENTION_DEBOUNCE_MS
+    );
+  },
+
+  @bind
+  _captureMentions() {
+    if (this.siteSettings.enable_mentions) {
+      const mentions = this._extractMentions();
+      this.onMention(mentions);
```

Comment:
It's a little bit unexpected that  `onMention` (singular) takes an array as param